Java/Queue/Queue.java

```diff
@@ -0,0 +1,118 @@
+package Java.Queue;
```

Comment:
@Khushi2402 
At the start of every program, we need to write a comment about the program and what problem it is solving. It really helps to know about the program.

In this case, we can write something like:
`Program to implement Queue`

Reply:
Okay Sure, I will add the necessary comments.